### PR TITLE
Homework 2

### PR DESCRIPTION
**Pull request by CherednichenkoNikita**

### CODE REVIEW

File: src/main/java/com/java_professional/cherednichenko/homework/lesson_2/task_2/LInkedListWithInteger.java [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+import java.util.LinkedList;
+
+public class LInkedListWithInteger {
+
+    public void getIntegerReverseList(LinkedList<Integer> linkedList) {
+        Comparator<Integer> comparator = Collections.reverseOrder();
+        linkedList.sort(comparator);
+        for (Integer integer : linkedList) {
+            System.out.println(integer);
+        }
```
Comment (ochedenis) [inline src/main/java/com/java_professional/cherednichenko/homework/lesson_2/task_2/LInkedListWithInteger.java:23]: you can just:
System.out.println(linkedList)

File: src/main/java/com/java_professional/cherednichenko/homework/lesson_2/task_2/LInkedListWithInteger.java [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+    }
+
+    public void getIntegerShuffleList(LinkedList<Integer> linkedList) {
+        Collections.shuffle(linkedList);
+        for (Integer integer : linkedList) {
+            System.out.println(integer);
+        }
```
Comment (ochedenis) [inline src/main/java/com/java_professional/cherednichenko/homework/lesson_2/task_2/LInkedListWithInteger.java:34]: you can just:
System.out.println(linkedList);

**Review by ochedenis — DISMISSED** [review]

File: src/main/java/com/java_professional/cherednichenko/homework/lesson_2/task_1/DoubleValues.java [hunk 3]
```diff
@@ -0,0 +1,29 @@
+package com.java_professional.cherednichenko.homework.lesson_2.task_1;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.ListIterator;
+import java.util.Scanner;
+
```
Comment (yuriy-tsarenko) [inline src/main/java/com/java_professional/cherednichenko/homework/lesson_2/task_1/DoubleValues.java:8]: Add the description of this task.

File: src/main/java/com/java_professional/cherednichenko/homework/lesson_2/task_1/DoubleValues.java [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+    public static void main(String[] args) {
+        List<String> stringList = new ArrayList<>();
+        Scanner scanner = new Scanner(System.in);
+        for (int i = 0; i < 5; i++) {
+            System.out.print("Input word:  ");
+            stringList.add(scanner.nextLine());
+        }
+        doubleValue(stringList);
+        for (String s : stringList) {
+            System.out.println(s);
```
Comment (yuriy-tsarenko) [inline src/main/java/com/java_professional/cherednichenko/homework/lesson_2/task_1/DoubleValues.java]: Shift your logic from main into special methods, main method body is too big

File: src/main/java/com/java_professional/cherednichenko/homework/lesson_2/task_1/DoubleValues.java [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+        }
+    }
+
+    public static void doubleValue(List<String> list) {
+        ListIterator<String> listIterator = list.listIterator();
+        while (listIterator.hasNext()) {
+            String s = listIterator.next();
+            listIterator.set(s + " " + s);
+        }
```
Comment (yuriy-tsarenko) [inline src/main/java/com/java_professional/cherednichenko/homework/lesson_2/task_1/DoubleValues.java]: It's bad practice, more better to code independent methods with return body

File: src/main/java/com/java_professional/cherednichenko/homework/lesson_2/task_2/LInkedListWithInteger.java (hunk 1 above)
Comment (yuriy-tsarenko) [inline src/main/java/com/java_professional/cherednichenko/homework/lesson_2/task_2/LInkedListWithInteger.java]: It's bad practice, more better to code independent methods with return body

File: src/main/java/com/java_professional/cherednichenko/homework/lesson_2/task_2/Main.java [hunk 6]
```diff
@@ -0,0 +1,19 @@
...
+        LinkedList<Integer> linkedList = new LinkedList<>();
+        Scanner scanner = new Scanner((System.in));
+        for (int i = 0; i < 10; i++) {
+            System.out.print("Input int: ");
+            linkedList.add(scanner.nextInt());
+        }
+        LInkedListWithInteger list = new LInkedListWithInteger();
+        list.getIntegerReverseList(linkedList);
+        list.getIntegerShuffleList(linkedList);
+        list.getMinimum(linkedList);
```
Comment (yuriy-tsarenko) [inline src/main/java/com/java_professional/cherednichenko/homework/lesson_2/task_2/Main.java]: Shift your logic from main into special methods, main method body is too big

**Review by yuriy-tsarenko — CHANGES_REQUESTED** [review]
Fix bugs according to my comments.
By the way, you have incorrect homework branch naming! Yo can read naming rules in slack.

**Pull request merged**